### PR TITLE
Remove 'filtered' and updated loookupFSC

### PR DESCRIPTION
**Pull request by ericrsmith35**
I removed my 'Filtered' components and updated the standard components as well as the README file.

### CODE REVIEW

File: flow_screen_components/lookupFSC/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (alexed1) [inline flow_screen_components/lookupFSC/.vscode/settings.json]: need to take this environment-specific stuff out. not everyone uses vscode, for example.

File: flow_screen_components/lookupFSC/README.md [hunk 2]
```diff
@@ -16,23 +16,63 @@ See the full set of installation options [here](/install.md).
 
 This component exposes the following attributes that can be set in Flow:
 
-label="Object Name" This is the actual object that will be looked up
+label="I1_Object Name" This is the actual object that will be looked up
```
Comment (alexed1) [inline flow_screen_components/lookupFSC/README.md]: what's the reason for adding the prefixes? they complicate the naming a little. Do they add value?

File: flow_screen_components/lookupFSC/README.md [hunk 3]
```diff
@@ -16,23 +16,63 @@ See the full set of installation options [here](/install.md).
...
+  I2_Display Which Field?           Subject
+  I3_Field Label                    Select Case
+  I4_Output Which Field as Value?   Id
+  I6_Filter on which field?         AccountId
+  I7_Filter Value                   {!vInputAcctId}
+  ---
+  O1_Output Value                   {!vCaseId}
+
+
+Example 2: Lookup an Account and a Contact on the same screen and only select from Contacts from the selected Account
```
Comment (alexed1) [inline flow_screen_components/lookupFSC/README.md]: so this is basically "Dependent Lookups", right? cool...

File: flow_screen_components/lookupFSC/README.md (hunk 2 above)
Comment (ericrsmith35) [reply flow_screen_components/lookupFSC/README.md]: I added the prefixes to identify input vs output parameters and to try and identify a logical sequence for assigning values

File: flow_screen_components/lookupFSC/README.md (hunk 3 above)
Comment (ericrsmith35) [reply flow_screen_components/lookupFSC/README.md]: Yes, optional dependant lookup

**Pull request merged**